addon/templates/components/lt-body.hbs

```diff
@@ -18,6 +20,7 @@
         {{else}}
           {{#each rows as |row|}}
             {{lt.row row columns
+              elementId=row.elementId
```

Comment:
Ah gotcha. Thanks for clarifying. I believe the `id` field is an alias to the `elementId`